keda/Chart.yaml

```diff
@@ -8,7 +8,7 @@ kubeVersion: ">=v1.17.0-0"
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.6.2
+version: 2.6.3
```

Comment:
Can you revert this please? We typically bump when we do a release

Reply:
Done.